SonarQube.Common/AnalysisConfig/AnalysisConfig.cs

```diff
@@ -33,6 +33,12 @@ public class AnalysisConfig
         ///  See https://jira.sonarsource.com/browse/SONARMSBRU-100 for details.</remarks>
         public string SonarRunnerWorkingDirectory { get; set; }
 
+        /// <summary>
+        /// Parent directory of the source files. 
+        /// </summary>
+        /// <remarks>SCM plugins like Git or TFVC use this to find .git or $tf subdirectories and provide annotations</remarks>
```

Comment:
The sources directory should contain .git / $tf subdirs. The SCM plugins currently do not search deeper.